src/pilot/templates/nic-configs/dpdk_5_port/nic_environment.yaml

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2018 Dell Inc. or its subsidiaries.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+resource_registry:
+  OS::TripleO::Controller::Net::SoftwareConfig: ./controller.yaml
+  OS::TripleO::DellCompute::Net::SoftwareConfig: ./compute.yaml
```

Comment:
Can you change the above line so that it references either compute-ovs-dpdk-mode1.yaml compute-ovs-dpdk-mode2.yaml since compute.yaml doesn't exist for this NIC config?